lib/Yam/Installation/Upgrade/SelectUpdatePage.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handle page that allows to select system/partition to be upgraded.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Installation::Upgrade::SelectUpdatePage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->{tbl_select_origin} = $self->{app}->table({id => "partition"});
```

Comment:
s/tbl_select_origin/tbl_partition_or_system_to_update/
we should avoid to use the internal ids, and always develop the POMs thinking from PoV of what the user can see in the screen.

Reply:
I'd rather the libyui id instead (tbl_partition), as in the case of https://openqa.suse.de/tests/11385576#step/remove_previous_repo/1 we would end-up with tbl_these_repositories_were_found_on_the_system_you_are_upgrading...

Reply:
you could shorten those strings, but I have seen really wrong one, Rainer has faced afir, and myself, internal fields usually don't help, long names are really clear, in this case `tbl_repositories_found` (if there is not other system you are upgrading), just an example, still follows the screen which help us with English and avoid our own personal choices for wording.